packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -235,6 +236,7 @@ const CreateActorRule: Rule = {
 			message: Message<Payloads.CreateActorCommon>
 		) => {
 			session.cacheInitializeActorMessage(message);
+			session.cacheAnimationCreationRequest(message);
```

Comment:
How is this actor creation message also an animation creation message?

Reply:
Animations can be created from either CreateAnimation or CreateFromPrefab messages.

---
In reply to: [364938287](https://github.com/microsoft/mixed-reality-extension-sdk/pull/475#discussion_r364938287) [](ancestors = 364938287)